.github/lexicon.txt

```diff
@@ -1617,6 +1617,7 @@ pvowner
 pvparameter
 pvparameters
 pvportmalloc
+pvportmallocstack
```

Comment:
Should vPortFreeStack also appear in the lexicon.txt?

Reply:
It is not reported as incorrect spell by the check.